docs/Cloud_integrations/Device_Integration_Center/kafka.md

```diff
@@ -43,13 +39,13 @@ Provide **basic information** such as the name and description, optionally enabl
 
     **Telemetry events** include data which is generated by the device. Events can originate from Objects, Object Instance, Resources or Resource Instances.
 
-![Select event triggers](images/events.png "Select event triggers")
+![Select event triggers](images/events.webp "Select event triggers")
 
 ## Configure Kafka parameters
 
 Specify the **Topic** and **Producer configuration** to set up the connection with a Kafka cluster or Azure Event Hub.
```

Comment:
Change to:
"To set up connection with a Kafka cluster or Azure Event Hub, specify the **Topic** and **Producer configuration**"